sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/v2/jdbc/JDBCTableCatalog.scala

```diff
@@ -135,7 +135,9 @@ class JDBCTableCatalog extends TableCatalog
       val schema = JDBCRDD.resolveTable(optionsWithTableName)
       JDBCTable(ident, schema, optionsWithTableName)
     } catch {
-      case _: SQLException => throw QueryCompilationErrors.noSuchTableError(ident)
+      case e: SQLException =>
+        logError("Failed to load table", e)
```

Comment:
`loadTable` is also used in normal cases, for example, testing if table is exist, may be `logWarn` is better

Reply:
Thanks, updated.